client/src/App.js

```diff
@@ -1,7 +1,12 @@
-import logo from './logo.svg';
+import React, { useEffect } from "react";
```

Comment:
As of React 17 we don't need to import React anymore, other than that looks good.